src/newComponents/viewer/LeadPreview/Preview/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+    _cs,
+    isValidUrl,
+} from '@togglecorp/fujs';
+import {
+    ImagePreview,
+} from '@the-deep/deep-ui';
+
+import {
+    MimeTypes,
+    isImageMimeType,
+    isDocMimeType,
+    isHTMLMimeType,
+    createUrlForGoogleViewer,
+} from './mimeTypes';
+
+import styles from './styles.scss';
+
+function isHttps(text?: string) {
```

Comment:
```suggestion
function isHttpsUrl(url?: string) {
```

Lets move this to utils